holder.yml

```diff
@@ -10,10 +10,106 @@ info:
     name: GitHub Source Code
     url: https://github.com/w3c-ccg/vc-api
 paths:
+  /credentials/{id}:
+    get:
+      tags:
+       - Credentials
+      summary: Gets a credential or verifiable credential by ID
+      operationId: getCredential
+      parameters:
+        - in: path
+          name: id
+          schema:
+            type: string
+            format: uuid
```

Comment:
Hrm, disagree w/ this only being able to be a UUID -- but we can deal w/ that in another PR. Just noting my objection to this one design choice.

Reply:
@mprorock we can take this change in our profile and leave this repo with `type` string where `<script> alert(1);</script>` is a spec legal identifier.

Reply:
> leave this repo with `type` string where `<script> alert(1);</script>` is a spec legal identifier.

Or, you know, do something more sane and re-use our existing transactionId/exchangeId regexes, which don't allow the attack you mention above:

https://github.com/w3c-ccg/vc-api/blob/main/components/parameters/path/TransactionId.yml